expenses/models.py

```diff
@@ -23,15 +23,15 @@ class Category(models.TextChoices):
     description = models.TextField(max_length=250, default='')
     category = models.CharField(max_length=32, choices=Category.choices, default=Category.OTHER)
     house_name = models.ForeignKey(House, on_delete=models.CASCADE)
-    amount = models.IntegerField(null=True, blank=True, validators=[MinValueValidator(0)])
+    amount = models.IntegerField(null=False, blank=False, validators=[MinValueValidator(0)])
```

Comment:
Can you please explain why you changed the Expense model?

Reply:
Also, I don't see here a new migrations files, did you run Django makemigrations ?

Reply:
I changed it because if it is set as it is set, then it gives an option not to put an amount into a new expense.
I'll run now Django makemigrations
@ronyyosef

Reply:
![image](https://user-images.githubusercontent.com/93659937/165341450-41023c87-a3d4-4568-88eb-50b7b44bcea2.png)
can I get help please? thanks!
@ronyyosef

Reply:
fixed